data/core/oos.md

```diff
@@ -28,501 +28,168 @@
-* greet
-    - action_greet_user
-* out_of_scope
-    - utter_out_of_scope
-* signup_newsletter{"email": "maxmeier@firma.de"}
-    - action_store_email
-    - slot{"email": "maxmeier@firma.de"}
-    - action_subscribe_newsletter
-    - slot{"subscribed": true}
-    - utter_awesome
-    - utter_confirmationemail
-    - utter_docu
-    - utter_ask_feedback
-
-## just newsletter (with email already) + confirm - already subscribed
+## just newsletter, don't continue + confirm
 * greet
     - action_greet_user
+* signup_newsletter
+    - utter_can_do
+    - subscribe_newsletter_form
+    - form{"name": "subscribe_newsletter_form"}
 * out_of_scope
     - utter_out_of_scope
-* signup_newsletter{"email": "maxmeier@firma.de"}
-    - action_store_email
-    - slot{"email": "maxmeier@firma.de"}
-    - action_subscribe_newsletter
-    - slot{"subscribed": false}
-    - utter_already_subscribed
-    - utter_docu
+    - utter_ask_continue_newsletter
+* deny
+    - utter_thumbsup
+    - action_deactivate_form
+    - form{"name": null}
     - utter_ask_feedback
 * affirm
     - utter_thumbsup
     - utter_anything_else
 
-## just newsletter (with email already) - already subscribed
+## just newsletter (with email already) + confirm
 * greet
     - action_greet_user
 * out_of_scope
     - utter_out_of_scope
 * signup_newsletter{"email": "maxmeier@firma.de"}
-    - action_store_email
-    - slot{"email": "maxmeier@firma.de"}
-    - action_subscribe_newsletter
-    - slot{"subscribed": false}
-    - utter_already_subscribed
-    - utter_docu
-    - utter_ask_feedback
-
-## just newsletter +confirm - already subscribed
-* greet
-    - action_greet_user
-* out_of_scope
-    - utter_out_of_scope
-* signup_newsletter
-    - utter_great
-    - utter_ask_email
-* enter_data{"email": "maxmeier@firma.de"} OR enter_data{"number":"1"}
-    - action_store_email
-    - slot{"email": "maxmeier@firma.de"}
-    - action_subscribe_newsletter
-    - slot{"subscribed": false}
-    - utter_already_subscribed
+    - utter_can_do
+    - subscribe_newsletter_form
+    - form{"name": "subscribe_newsletter_form"}
+    - form{"name": null}
     - utter_docu
     - utter_ask_feedback
 * affirm
     - utter_thumbsup
     - utter_anything_else
 
-## just newsletter +confirm - already subscribed
+## just newsletter (with email already)
 * greet
     - action_greet_user
-* signup_newsletter
-    - utter_great
-    - utter_ask_email
 * out_of_scope
     - utter_out_of_scope
-    - utter_possibilities
-    - utter_ask_email
-* enter_data{"email": "maxmeier@firma.de"} OR enter_data{"number":"1"}
-    - action_store_email
-    - slot{"email": "maxmeier@firma.de"}
-    - action_subscribe_newsletter
-    - slot{"subscribed": false}
-    - utter_already_subscribed
+* signup_newsletter{"email": "maxmeier@firma.de"}
+    - utter_can_do
+    - subscribe_newsletter_form
+    - form{"name": "subscribe_newsletter_form"}
+    - form{"name": null}
     - utter_docu
     - utter_ask_feedback
-* affirm
```

Comment:
why?

Reply:
Same as before, this is actually the story that starts in [line 119](https://github.com/RasaHQ/rasa-demo/pull/302/files/fbefffb9ca16d37a7e9d6b1180e566fcd69e3797#diff-065c30c4a8d385836f4ce60741d573d0L119), you can tell because oos comes before signup newsletter